scripts/clippy.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Install Cargo Clippy for the specified toolchain
+rustup component add clippy --toolchain 1.76.0-x86_64-unknown-linux-gnu
+
```

Comment:
```suggestion
```

We shouldn't include this here since it will go out of sync with the rust toolchain specified elsewhere